react/components/ShippingSimulator/components/ShippingTable.js

```diff
@@ -1,12 +1,11 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { FormattedMessage } from 'react-intl'
+import { withCssHandles } from 'vtex.css-handles'
 
 import ShippingTableRow from './ShippingTableRow'
 
-import styles from '../shippingSimulator.css'
-
-export default class ShippingTable extends Component {
+class ShippingTable extends Component {
```

Comment:
(Could have changed to function component)

Reply:
not the purpose of this PR